src/components/quiz/QuizQuestions.tsx

```diff
@@ -1,9 +1,54 @@
 import React from "react";
+import { useState } from "react";
```

Comment:
These should be the same import:
```suggestion
import React, { useState } from "react";
```